docs/conf.py

```diff
@@ -55,8 +50,12 @@
 copyright = u'2013-%s, David R. MacIver' % datetime.datetime.utcnow().year
 author = u'David R. MacIver'
 
-version = __version__
-release = __version__
+_d = {}
+with open(os.path.join(os.path.dirname(__file__), '..', 'src',
+                       'hypothesis', 'version.py')) as f:
+    exec(f.read(), _d)
```

Comment:
Mmm. We already have one copy of this code in `setup.py`, do we need to duplicate it?

Reply:
For only two uses in independent scripts, I think it's cleaner to duplicate it than to share.

Reply:
It actually appears in a third place FWIW, which is in hypothesistooling.py

Reply:
The basic problem though is that if we could import the snippet we could just import the `__version__` instead, and #722 shows some of the many problems that can cause.

I still think this is the least contorted way of doing the right thing, but would be delighted to take concrete suggestions!

Reply:
I'm OK to leave it as is.